pkg/sql/codegen/couler/codegen.go

```diff
@@ -38,10 +38,28 @@ func Run(programIR ir.SQLProgram, session *pb.Session) (string, error) {
 		defaultDockerImage = os.Getenv("SQLFLOW_WORKFLOW_STEP_IMAGE")
 	}
 	for _, sqlIR := range programIR {
-		sqlStmt := &sqlStatement{
-			OriginalSQL: sqlIR.GetOriginalSQL(), IsExtendedSQL: sqlIR.IsExtended(),
-			DockerImage: defaultDockerImage}
-		r.SQLStatements = append(r.SQLStatements, sqlStmt)
+		switch i := sqlIR.(type) {
+		case *ir.StandardSQL:
```

Comment:
What to do for `PredictStmt` and `ExplainStmt`?

Reply:
I do not find `ExplainStmt`, but find `AnalyzeStmt` and add them into the `switch`.